fullcontact/api/person_api.py

```diff
@@ -1,13 +1,20 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module serves the class for making
+FullContact Person Enrich API requests.
+"""
+
 from .base.enrich_base import EnrichBase
 from ..response.person_response import PersonEnrichResponse
 from ..schema.person_schema import PersonSchema
 
 
-class PersonClient(EnrichBase):
+class PersonApi(EnrichBase):
     r"""
     Class that provides methods to perform
     Person Enrich operations, with MultiField Capabilities.
     """
-    enrich_endpoint = "person.enrich/"
+    _enrich_endpoint = "person.enrich/"
```

Comment:
I didn't realize the trailing slash worked for APIs, I've always done `person.enrich`...

Reply:
😄  I always use a trailing slash!

Reply:
Removed.